how-to/about-how-to.md

```diff
@@ -1,62 +1,57 @@
-# The how-to article
+# The How To article
 
-## When do I need a how-to article?
+How To articles take the reader through a series of steps required to solve a specific problem.
+They explain in detail how to do a specific task.
+Don't use How To articles to teach concepts, but to prescribe a set of sequential steps readers must do to be able to use a feature.
 
-How-to articles take the reader through a series of steps required to solve a specific, real-world problem.
-Think of them as recipes for getting stuff done!
+How To articles are often confused with tutorials. How To articles are problem-oriented, while tutorials are learning-oriented.
 
-Sometimes you need to explain in detail how to do a specific task to a user.
-You don't need to teach them concepts, but you do need to prescribe a set of sequential steps they must do in that order otherwise they won't be able to use a feature properly.
+Readers that have experience with the product and need a specific question answered use How To articles.
+New users who want to solve a problem instantly can also benefit, provided they are well-written and state any prerequisite knowledge required to complete the task.
 
-How-to articles are often confused with tutorials, which is not the case as they are _problem_-oriented not _learning_-oriented.
+## Content of your How To article
 
-How-to articles are used by those who have experience with the product and need a specific question answered.
-New users who want to solve a problem instantly can also benefit from how-to articles provided they are well-written and clearly state any prerequisite knowledge required to complete the how-to.
+### The Overview section
 
-## Content of your how-to article
+Summarize what the reader will achieve by reading the explanation article.
 
-### About the "Overview" section
+* Are you writing for developers or for managers?
+* Are you writing for people who have a certain problem to solve?
+* Are you writing for a particular industry or market segment?
 
-Summarize what the reader will get from reading the how-to articles.
+Readers should already have read a concept about the topic in an About article.
+Ensure you link to the relevant About article in the overview.
 
-Creating a good overview comes down to thinking about who is going to be using it:
+### The Before you start section
```

Comment:
Maybe change to:

`### The "Before you start" section` 

per the above idea. This will be the last time I suggest the pattern in my review.